### ISSUE TITLE
rust issue

### ISSUE THREAD

**Issue by lucize** [lmahmutov/SGW-Homeassistant#40]
I guess we are stuck for now with 2022.6.x release as 2022.7.x needs orjson that seems to need rust and we still don't have that in OpenWrt

**Comment by lmahmutov** [comment]
Openwrt 23 added full rust support )

**Comment by lucize** [comment]
yes, but I'm a bit bizi with life now and since I migrated to docker it's not so urgent for me now, but it's on the todo list
